lib/docs/src/tag.stories.tsx

```diff
@@ -1,24 +1,54 @@
 import { Meta } from "@storybook/react";
 import { Fragment } from "react";
-import { DivPx, Tag } from "../../core/src";
+import { Button, DivPx, Tag } from "../../core/src";
+import { Utils } from "./utils";
 
-export default {
-	title: "Draft/Tag",
+const colors: string[] = [
+	"red",
+	"yellow",
+	"green",
+	"blue",
+	"indigo",
+	"purple",
+	"pink",
+	"gray",
+];
+
+const meta: Meta = {
+	title: "Components/Tag",
 	component: Tag,
-} as Meta;
-
-export const Primary = (): JSX.Element => {
-	const colors: string[] = [
-		"red",
-		"yellow",
-		"green",
-		"blue",
-		"indigo",
-		"purple",
-		"pink",
-		"gray",
-	];
+	argTypes: {
+		children: Utils.arg("text"),
+		color: Utils.arg(colors),
+	},
+};
+
+Utils.page.component(meta, { sticky: true, shots: [] });
+
+export default meta;
+
+interface Props {
+	children?: string;
+	color?: string;
+}
+
+export const Primary = (props: Props): JSX.Element => {
+	return (
+		<Tag color={(Tag.colors as any)[props.color ? props.color : "gray"]}>
+			{props.children ?? "Default"}
+		</Tag>
+	);
+};
 
+export const Basic = (): JSX.Element => {
+	return <Tag color={Tag.colors.green}>Tag</Tag>;
+};
+
+Utils.desc(Basic)(`
+Tags are used to provide additional information about something. To begin, you need to provide a color from Tag.color and some information as string via children.
+`);
```

Comment:
> Tags are used to provide additional information about something.

We don't need to repeat the usage of the component here. It's already in the description at the top of the page, right above this Basic section.

Reply:
> To begin, you need to provide a color from Tag.color and some information as string via children.

"Tag.color" should be "Tag.colors" I think

"some information as string via children." -> saying "information" is too specific. Should be something like "and a label via the "children" prop"